### PR TITLE
Admin panel - QGIS projects view: overall UI improvements

### LINKED ISSUE #3786: Show color coding also for selected project
In LMWC projects page you introduced a very useful and nice page to show details about projects with color coding to warn user about possible issues. 

But users can select a project in the page (and there is always a selected one on page load), and for selected rows te color coding is not visible anymore.

image

image

### PR DESCRIPTION
**Pull request by mdouchin**
* Use the datatable responsive extension to automatically hide some column if the width of the screen is not wide enough
* Remove the right panel to gain more place
* Remove the blue background-color fix #3786 of the last clicked line 
* Remove the table colours legend previously shown at the bottom of the table and replace it with a modal popup accessible via a new button displayed in the top right side of the table.
* Overall code improvement and simplification

Funded by 3liz

### CODE REVIEW

File: lizmap/modules/admin/templates/project_list_zone.tpl [hunk 1]
```diff
@@ -15,43 +17,46 @@
 </div>
 {/if}
 
-{if $hasSomeProjectsNotDisplayed}
-<div>
-    {@admin.project.not.displayed@}
```
Comment (Gustry) [inline lizmap/modules/admin/templates/project_list_zone.tpl:20]: This string is still in the source code. I guess we want to remove it or we should just rephrase ?

**Review by Gustry — APPROVED** [review]
I saw it live many times, it's way better ⭐

File: lizmap/modules/admin/templates/project_list_help.tpl [hunk 2]
```diff
@@ -0,0 +1,100 @@
+<div id="lizmap_project_list_help" class="modal hide fade" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+    <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+        <h3>{@admin.project.modal.title@}</h3>
+    </div>
+    <div class="modal-body" style="background-color: rgb(239, 239, 239);">
+        <style type="text/css" scoped>
```
Comment (nworr) [inline lizmap/modules/admin/templates/project_list_help.tpl:7]: `scoped` is deprecated / unsupported , no ?

https://caniuse.com/style-scoped

File: lizmap/modules/admin/templates/project_list_zone.tpl [hunk 3]
```diff
@@ -211,21 +196,65 @@
...
+                </ul>
             </td>
+            {else}
+            <td></td>
+            {/if}
+
+            <!-- Hidden QGIS logs -->
+            <td class="lizmap-project-qgis-log">
+                {if !empty($p['qgis_log'])}
+                <div>
```
Comment (laurentj) [inline lizmap/modules/admin/templates/project_list_zone.tpl:252]: `<pre>` should be used instead a `<div>` here.

File: lizmap/www/assets/css/admin.css [hunk 4]
```diff
@@ -89,65 +89,85 @@ fieldset {
...
+
+/*
+======
+Admin QGIS project lists
+======
+*/
 div#lizmap_project_list_container {
-    display: flex;
-    flex-wrap: wrap;
+    /* display: flex;
```
Comment (laurentj) [inline lizmap/www/assets/css/admin.css:99]: remove the comment

File: lizmap/modules/admin/templates/project_list_zone.tpl (hunk 1 above)
Comment (mdouchin) [reply lizmap/modules/admin/templates/project_list_zone.tpl:20]: I used a jMessage to display a warning at the top of the page

**Pull request merged**